client/cluster-client/src/main/kotlin/uk/ac/ebi/biostd/client/cluster/api/ClusterClient.kt

```diff
@@ -0,0 +1,95 @@
+package uk.ac.ebi.biostd.client.cluster.api
+
+import arrow.core.Try
+import com.jcraft.jsch.JSch
+import com.jcraft.jsch.Session
+import ebi.ac.uk.coroutines.waitUntil
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.withContext
+import mu.KotlinLogging
+import uk.ac.ebi.biostd.client.cluster.common.JobResponseParser
+import uk.ac.ebi.biostd.client.cluster.common.JobSubmitFailException
+import uk.ac.ebi.biostd.client.cluster.model.Job
+import uk.ac.ebi.biostd.client.cluster.model.JobSpec
+import java.time.Duration.ofSeconds
+
+private const val CHECK_COMMAND = "bjobs -o STAT -noheader %s"
+private const val DONE_STATUS = "DONE"
+private const val SUBMIT_COMMAND = "bsub -o %s/%%J_OUT -e %s%%J_IN"
+
+private val logger = KotlinLogging.logger {}
+
+class ClusterClient(
```

Comment:
such a pain that this file has not been detected as a rename, basically we are lossing all the changes history